core/types/graph.cpp

```diff
@@ -337,16 +337,16 @@ void GraphEdge::_notification(int p_what) {
 }
 
 void GraphEdge::_bind_methods() {
-	ClassDB::bind_method(D_METHOD("get_vertex_a"), &GraphEdge::get_vertex_a);
-	ClassDB::bind_method(D_METHOD("get_vertex_b"), &GraphEdge::get_vertex_b);
+	ClassDB::bind_method(D_METHOD("get_a"), &GraphEdge::get_a);
```

Comment:
maybe start/end? might be good?

Reply:
`from/to` implies direction, which is not always the case for an edge (associative edges), so this may lead to confusion. You can determine whether an edge has a direction by calling `is_directed()`

For associative edges, the following statements are equivalent:
```gdscript
graph.has_edge(a, b)
graph.has_edge(b, a)
```

Reply:
It seems a bit weird with a, b, could we go for any other name?

Reply:
Without implying direction, `first/second` comes to mind. This is how edges are accessed internally (via `Pair` keys, which also has `first/second` members).

Reply:
get_first() and get_last() probably?

Reply:
`last` sounds like there's a collection to choose from, however it's shorter. Yet `a/b` is even shorter. 😛

If you look at Godot naming, the `SegmentShape2D` also has `a` and `b` to denote endpoints.

Joints have both `node_a` and `node_b` properties.

That's why I prefer `a/b` because it's the naming convention Godot currently uses. I'm open to other opinions, though.